wiki/Tournaments/OZT/2018/en.md

```diff
@@ -0,0 +1,114 @@
+# osu! Z-Tournament 2018
+![osu!Z-Tournament 2018](logo.png)
+
+- The **osu! Z-Tournament 2018**, is a 4v4 international osu!standard tournament.
```

Comment:
Can you rewrite these into a paragraph?
Example: <https://osu.ppy.sh/help/wiki/Tournaments/TWC/2018>